src/Lean/AddDecl.lean

```diff
@@ -8,6 +8,7 @@ module
 prelude
 public import Lean.Meta.Sorry
 public import Lean.Util.CollectAxioms
+import Lean.Compiler.Old
```

Comment:
Hmm this is probably a fishy import. We certainly don't want to depend on the old compiler in a very central module